VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineActivity.java

```diff
@@ -74,6 +72,10 @@ protected void onPostCreate(Bundle savedInstanceState) {
                     Bundle bundle = new Bundle();
                     bundle.putSerializable(Router.EXTRA_ENROLLMENT, courseData);
                     bundle.putString(Router.EXTRA_COURSE_COMPONENT_ID, courseComponentId);
+                    if (getIntent() != null) {
```

Comment:
I don't think the `Intent` can be null in the normal course of things, so this check is not needed.